testing/driver/sd_bench/Makefile

```diff
@@ -1,5 +1,5 @@
 ############################################################################
-# apps/testing/sd_bench/Makefile
+# apps/testing/driver/sd_bench/Makefile
```

Comment:
benchmark shold move to apps/benchmark

Reply:
We talked about this these sd tests in https://github.com/apache/nuttx-apps/pull/2931.. looks like good test case for sd card drivers in terms of benchmark and stress-test? But no strong opinion on this one maybe benchmark is better location for this sd_bench :-)

Reply:
test should report pass/fail clearly, so we can run it and check the result automatically. sd_bench doesn't have this key characteristic.

Reply:
Yup, again need of testing utils and tests cases separation?